softee220/as_build_vector_db.py

```
@@ -0,0 +1,159 @@
+    # db 초기화
+    dbclient, collection = init_db("./chroma_db")
+
+    csv_path = "./naverent_movie_data.csv" # 데이터 가져다 쓸 경로 지정
+    rows = load_rows_from_csv(csv_path) # 처리할 문서 데이터 메모리로 불러오기
+
+    doc_id = 0
+    
+    for row_idx, title, desc, text in rows:
+        # 기사 길이에 따라 청크 크기 조절
+        chunks = chunk_text(text, chunk_size=500, chunk_overlap=50) # chunking
+        for idx, chunk in enumerate(chunks):
```

Ruff (0.13.2)

Loop control variable `idx` not used within loop body

(B007)